ml-agents/setup.py

```diff
@@ -61,7 +62,7 @@ def run(self):
         "Pillow>=4.2.1",
         "protobuf>=3.6",
         "pyyaml",
-        "tensorflow>=1.7,<2.1",
+        "tensorflow>=1.7,<3.0",
```

Comment:
I did have to add `six>=1.12.0` to setup.py for tf>2.1.0

Reply:
yeah, I'm not sure what to do about this. pip _should_ be able to figure it out; it's not really our requirement, just a transitive one.